src/transformers/trainer.py

```diff
@@ -193,6 +194,15 @@
     IS_SAGEMAKER_MP_POST_1_10 = False
 
 
+skip_first_batches = None
+if is_accelerate_available():
+    from accelerate import __version__ as accelerate_version
+
+    # Change this to >= 0.16 once the release is out
+    if version.parse(accelerate_version) > version.parse("0.15"):
+        from accelerate import skip_first_batches
+
```

Comment:
```suggestion
    if version.parse(accelerate_version) > version.parse("0.15"):
        from accelerate import skip_first_batches
    else:
        skip_first_batches = None
```

no?

Reply:
It's set before the first test of accelerate availability as it also needs to be None if accelerate is not available.

Reply:
ah, oops, I missed it! sorry about that.

Reply:
No worries!